alfresco-transform-tika/alfresco-transform-tika/LICENSES.md

```diff
@@ -1,5 +1,4 @@
 ### Licenses
 
-* Tika is from Apache. See the license at http://www.apache.org/licenses/LICENSE-2.0 or the 
-[Apache 2.0.txt](https://github.com/Alfresco/acs-community-packaging/blob/master/distribution/src/main/resources/licenses/3rd-party/Apache%202.0.txt) 
-file placed in the root directory of the docker image.
+* Tika is from Apache. See the licence at http://www.apache.org/licenses/LICENSE-2.0 or the [Apache 2.0.txt](https://github.com/Alfresco/alfresco-community-repo/blob/master/packaging/distribution/src/main/resources/licenses/3rd-party/Apache%202.0.txt) file placed in the root directory of the docker image.
+* Exiftool is from Phil Harvey. See licence at https://exiftool.org/#license and Perl at https://dev.perl.org/licenses/ (https://dev.perl.org/licenses/gpl1.html & https://dev.perl.org/licenses/artistic.html) or the [GNU General Public License Version 3.txt](https://github.com/Alfresco/alfresco-community-repo/blob/master/packaging/distribution/src/main/resources/licenses/3rd-party/Perl%20GNU%20License.txt) and [Perl Artistic License.txt](https://github.com/Alfresco/alfresco-community-repo/blob/master/packaging/distribution/src/main/resources/licenses/3rd-party/Perl%20Artistic%20License.txt) files placed in the root directory of the docker image.
```

Comment:
Only link to Perl Artistic lincense. Remove "or" part. 

If still needed then also replace https://github.com/Alfresco/alfresco-community-repo ... with link to https://github.com/Alfresco/alfresco-transform-core ?

Reply:
The licence file is no in transform-core. It's still in the repo project.

GNU licences removed.

Reply:
See previous comment. Why not link to:

https://github.com/Alfresco/alfresco-transform-core/pull/384/files#diff-8944d537efc1e569d29a2ed0f0586340b8270ec3d9b5423c6e32108dbde1524d (but using "master" reference, pending merge).

Reply:
The licence files are in the top level of each *-boot project so that they are copied into the docker image.

Reply:
Thanks for fixing /licenses/ folder. 

Just this url looks wrong ?

https://github.com/Alfresco/alfresco-community-repo/blob/master/packaging/distribution/src/main/resources/licenses/3rd-party/Perl%20Artistic%20License.txt

Reply:
Thanks for fixing.